glossary.rst

```diff
@@ -219,6 +230,29 @@ Glossary
   Ubiquitous Language
     TODO
 
+  Result Backend
+    TODO
+
+  Celery Master
+    TODO
+
+  Celery Worker
+    TODO
+
+  Celery Multi
+    TODO
+
+  Flower
```

Comment:
we need better alternative of flower, based on asgi.

Reply:
I'm not sure we necessarily need to push for killing flower (competition is good, though!). Flower is running on an event loop and once it cuts over to py3, would it be all that hard to simply swap it to asyncio and support asgi? Can tornado support asgi (in which case less code change)?

Reply:
We're going to replace our entire observability stack.
The plan is to use open telemetry.
Flower would need a rewrite in any case.

Reply:
**good plan**

Reply:
OpenTelemetry is a great long term target (it is likely the ideal! And also curious about the timeline with the draft W3C standard) but even their docs recommend not using it in production because it's currently in beta.

Depending on our timing for implementation, this may not be an issue.

I'd like to suggest that if we are going to put design implementation details in this document that instead of explicitly calling that out we say OpenTelemetry (beta, evolution of OpenTracing & OpenCensus), OpenTracing (older but stable), or other framework.

Reply:
I'm not going to write anything about observability in this document except to mention that this is one of our goals.